### DISCUSSION TITLE
网上取名那些事

### DISCUSSION THREAD

**Discussion by kouhe3** [kouhe3/discussion#5] [Ideas]
网上起名字。
关于 昵称，用户名，人名， ID , 社交网络统一识别号 (个人全球统一标识档案， Gravater ).
的一些感想。

**Comment by kouhe3** [comment]
通常我建议那些还没有开始混各大社交网站的同学想出一个好的英文用户名，然后在以后的日子里注册网站的时候都使用这一个用户名。
我认为一个好的用户名是：
容易读，容易记，不是太长，独特。

我就习惯在多个社交网站使用 kouhe3 这个名字，如果什么地方的 kouhe3 被别人抢了，我会很不舒服。

**Reply by GrassBlock1** [reply]
快给我起一个（不是

**Reply by kouhe3** [reply]
GRASBLOK ? GrassBlockie ? iGrassBlock?

**Reply by GrassBlock1** [reply]
> GRASBLOK ? GrassBlockie ? iGrassBlock?

（（（

**Comment by kouhe3** [comment]
起名字的时候，我们通常都希望起一个不会和别人搞混的名字。
但事实并不像我们想的那样。
起的用户名经常有人占,
或者是因为别的原因没有办法使用。
于是我们不得不在有些时候放弃我们喜欢的名字。
我们都想要一个统一标识号，
在网站上别人看到账户名的时候能知道那就是自己。
但是当用户名无法使用的时候，要怎么办呢？

**Comment by kouhe3** [comment]
也许重要的并不是那个注册网站时可以自定义的用户名, 
有更重要的依据让别人可以知道那就是你。

**Comment by kouhe3** [comment]
### 论昵称和简称的作用

用户名有时候会很长，或者是有数字，或者别的一些原因。全部读出来，写出来会很麻烦，也不容易记。
于是朋友可能更喜欢叫你的简称或者昵称。
简称可能是用户名的前几个字，昵称可能是与用户名无关的称呼。
在交流中，可能更多的提及昵称而不是用户名。
还有外号。

**Comment by kouhe3** [comment]
如果喜欢的用户名不能使用，一个退一步的方法是，在原有的用户名的基础上增加或者删去几个字符，比如
开头加上 i , the .
结尾加上 数字, 或者 er , ies , en.
这样，在用户名可用的同时尽量不失去标志作用。

**Comment by kouhe3** [comment]
有时为了匿名，可能需要
故意使用一些容易造成误解的用户名 (file_transfer) ;
或者是随机字符串 (nlosgwetoagbdunnoi) ;
或者是看起来很一般的与统一标识符没有关系的名字 (Alan Gardner , tutorbat) ;

**Reply by GrassBlock1** [reply]
Blocker  （（（

**Comment by kouhe3** [comment]
别人称呼你的方式不一定是你定的，他们叫你的昵称可能各不一样。

**Comment by kouhe3** [comment]
决定用户名的时候，考虑好以下几点：
用户名独特性。
用户名被占用时的退步；
可能会用到编号；
用户名简称。
